modules/core/src/copy.cpp

```diff
@@ -414,6 +414,46 @@ void Mat::copyTo( OutputArray _dst, InputArray _mask ) const
         copymask(ptrs[0], 0, ptrs[2], 0, ptrs[1], 0, sz, &esz);
 }
 
+// just 1 byte is valid
+#define CANT_APPLY_MEMSET int(0xDEADBEEF)
```

Comment:
Avoid magic constants.
- return "bool" value
- add "output" parameter reference which will be filled.

(keep source code and binaries clean)

Reply:
I agree. ( nice suggestion )
Your suggestion make code better clean.